inc/commonitiltask.class.php

```diff
@@ -1829,7 +1829,8 @@ public static function getTaskList($status, $showgrouptickets, $start = null, $l
          if (isset($_SESSION['glpigroups']) && count($_SESSION['glpigroups'])) {
             $prep_req['WHERE'][self::getTable() . '.groups_id_tech'] = $_SESSION['glpigroups'];
          } else {
-            return false;
+            // Return empty iterator result
+            return new DBmysqlIterator($DB);
```

Comment:
You can handle this the same way it is done for tickets:
```suggestion
            $prep_req['WHERE'][] = 0;
```